### PR TITLE
New plugin API

### PR DESCRIPTION
**Pull request by eile**

### CODE REVIEW

**Comment by eile** [comment]
Retest this please.

**Comment by eile** [comment]
Ping: this blocks latest.

File: livre/eq/Client.cpp [hunk 1]
```diff
@@ -35,17 +35,10 @@
...
-    Impl()
-    {
-    }
-
-    ~Impl()
-    {
-        DataSource::unloadPlugins();
-    }
+    Impl() {}
+    ~Impl() {}
```
Comment (rdumusc) [inline livre/eq/Client.cpp]: rm ?

**Comment by eile** [comment]
commit renamed

**Pull request merged**